src/main/java/edu/kis/vh/nursery/stack/IntStack.java

```diff
@@ -0,0 +1,19 @@
+package edu.kis.vh.nursery.stack;
+
+public interface IntStack {
```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.